apps/admin/frontend/src/components/import_cvrfiles_modal.tsx

```diff
@@ -310,6 +315,10 @@ export function ImportCvrFilesModal({ onClose }: Props): JSX.Element | null {
     );
   }
 
+  if (currentState.state === 'loading') {
+    return <Modal content={<Loading>Loading CVRs</Loading>} />;
```

Comment:
Note that I'm intentionally not specifying an `onOverlayClick` prop for saving/loading modals so that clicks outside them don't accidentally close them. This came up a few times during the SLI test as a source of confusion. Clicking outside the modal would dismiss it, but the relevant operation would continue behind the scenes, leaving the machine sluggish without any clear indication as to why it was sluggish.